packages/cmr-client/src/getUrl.ts

```diff
@@ -1,56 +1,67 @@
-function getCmrHost(cmrEnvironment?: string): string {
+/**
+ * Get host to use for CMR requests.
+ *
+ * @param {string} [cmrEnvironment]
+ *  CMR environment to use for requests. Can be a CMR environment "logical"
+ *  name or a full custom host name.
+ * @returns {string} - CMR host
+ * @private
+ **/
+export function getCmrHost(
+  cmrEnvironment: string | undefined = process.env.CMR_ENVIRONMENT
+): string {
+  if (!cmrEnvironment) {
+    throw new TypeError(`Invalid CMR environment: ${cmrEnvironment}`);
+  }
+  let cmrHost;
   switch (cmrEnvironment) {
     case 'OPS':
-      return 'cmr.earthdata.nasa.gov';
+      cmrHost = 'cmr.earthdata.nasa.gov';
+      break;
     case 'UAT':
-      return 'cmr.uat.earthdata.nasa.gov';
+      cmrHost = 'cmr.uat.earthdata.nasa.gov';
+      break;
     case 'SIT':
-      return 'cmr.sit.earthdata.nasa.gov';
+      cmrHost = 'cmr.sit.earthdata.nasa.gov';
+      break;
     default:
-      throw new TypeError(`Invalid CMR environment: ${cmrEnvironment}`);
+      cmrHost = cmrEnvironment;
   }
+  return cmrHost;
 }
 
 export function getIngestUrl({
-  host,
-  cmrEnv = process.env.CMR_ENVIRONMENT,
+  cmrEnv,
   provider,
 }: {
-  host?: string,
   cmrEnv?: string,
   provider: string,
 }): string {
-  return `https://${host ?? getCmrHost(cmrEnv)}/ingest/providers/${provider}/`;
+  return `https://${getCmrHost(cmrEnv)}/ingest/providers/${provider}/`;
 }
 
 export function getSearchUrl({
-  host,
```

Comment:
I checked and I didn't see these `host` arguments being passed in from anywhere and it ultimately feels like having them introduces more confusion if they're not being used

Reply:
Agreed, however that makes this a breaking change re: the cmr-client's API.  As it's a public package, we  should note it in the changelog as such.

Reply:
I think this still needs addressed

Reply:
With the latest changes, the `host` argument to these functions is no longer removed. Unless I'm misunderstanding?

Reply:
Right you are, it's odd that this isn't "Outdated".       Resolving.